src/scss/sections/_header.scss

```diff
@@ -0,0 +1,38 @@
+.img-logo-adaptiv {
+  max-width: 100%;
+  height: auto;
+  @include adaptiv-img(155, 58);
+}
+
+.phone-number {
+  padding-left: 12px;
+}
+
+.color-accent {
+  color: $color-accent;
+}
```

Comment:
Move to `base.scss` or another file, which response to helpers or utils classes.